### PR TITLE
Document Cron Builds

### LINKED ISSUE #39: Support cron builds
Cron builds would allow to schedule the execution of a build on a specified branch, at fixed intervals, independently of whether any commits were pushed to the repository. 
Configuring a cron build from cirrus-ci.com would require to define the interval (hourly, daily, weekly or a custom value) and the branch (by default the branch marked as "default" in GitHub should be used).

Cron builds send notifications as any other builds.

### PR DESCRIPTION
**Pull request by fkorotkov**
Fixes #39

### CODE REVIEW

File: docs/guide/writing-tasks.md [hunk 1]
```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
     the variable by themselves and editing both the encrypted variable and repo-owner condition
     in the `.cirrus.yml` file.
 
+## Cron Builds
+
+It is possible to configure invocations of re-occurring builds via a well-known Cron expressions. Cron builds can be
```
Comment (RDIL) [inline docs/guide/writing-tasks.md]:
```suggestion
It is possible to configure invocations of re-occurring builds via the well-known Cron expressions. Cron builds can be
```

File: docs/guide/writing-tasks.md [hunk 2]
```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
...
+configured on a repository's settings page instead of configuring them in `.cirrus.yml`.
```
Comment (RDIL) [inline docs/guide/writing-tasks.md]:
```suggestion
configured in a repository's settings page, or in the repository's `.cirrus.yml`.
```

File: docs/guide/writing-tasks.md [hunk 3]
```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
...
+
+It's possible to configure several cron builds with unique `names` which will be avaialbale via `CIRRUS_CRON` [environment variable](#environment-variables).
```
Comment (RDIL) [inline docs/guide/writing-tasks.md]:
```suggestion
It's possible to configure several cron builds with unique `names` which will be available via `CIRRUS_CRON` [environment variable](#environment-variables).
```

File: docs/guide/writing-tasks.md [hunk 4]
```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
...
+Each cron build should specify *branch* to trigger new builds for and a cron expression compatible with Quarts. Please use
+[this generator](https://www.freeformatter.com/cron-expression-generator-quartz.html) to generate/validate your expressions.
+
+**Note:** Cron Builds are using UTC timezone.
```
Comment (RDIL) [inline docs/guide/writing-tasks.md]:
```suggestion
**Note:** Cron Builds are timed with the UTC timezone.
```

File: docs/guide/writing-tasks.md [hunk 5]
```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
     the variable by themselves and editing both the encrypted variable and repo-owner condition
     in the `.cirrus.yml` file.
 
+## Cron Builds
+
+It is possible to configure invocations of re-occurring builds via a well-known Cron expressions. Cron builds can be
+configured on a repository's settings page instead of configuring them in `.cirrus.yml`.
+
+It's possible to configure several cron builds with unique `names` which will be avaialbale via `CIRRUS_CRON` [environment variable](#environment-variables).
+Each cron build should specify *branch* to trigger new builds for and a cron expression compatible with Quarts. Please use
```
Comment (RDIL) [inline docs/guide/writing-tasks.md]:
```suggestion
Each cron build should specify *branch* to trigger new builds for and a cron expression compatible with Quarts. You can use
```

File: docs/guide/writing-tasks.md (hunk 2 above)
Comment (fkorotkov) [reply docs/guide/writing-tasks.md]: There is no way to configure Cron Builds in `.cirrus.yml`

Comment (RDIL) [reply docs/guide/writing-tasks.md]: the way it was worded made it sound like it was possible

File: docs/guide/writing-tasks.md [hunk 6]
```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
     the variable by themselves and editing both the encrypted variable and repo-owner condition
     in the `.cirrus.yml` file.
 
+## Cron Builds
+
+It is possible to configure invocations of re-occurring builds via the well-known Cron expressions. Cron builds can be
+configured on a repository's settings page instead of configuring them in `.cirrus.yml`.
```
Comment (fkorotkov) [inline docs/guide/writing-tasks.md]: @RDIL how about this:

```suggestion
configured on a repository's settings page (not in `.cirrus.yml`).
```

Comment (RDIL) [reply docs/guide/writing-tasks.md]: that works

File: docs/guide/writing-tasks.md [hunk 7]
```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
     the variable by themselves and editing both the encrypted variable and repo-owner condition
     in the `.cirrus.yml` file.
 
+## Cron Builds
+
+It is possible to configure invocations of re-occurring builds via the well-known Cron expressions. Cron builds can be
+configured on a repository's settings page (not in `.cirrus.yml`).
+
+It's possible to configure several cron builds with unique `names` which will be available via `CIRRUS_CRON` [environment variable](#environment-variables).
+Each cron build should specify *branch* to trigger new builds for and a cron expression compatible with Quarts. You can use
```
Comment (gspencergoog) [inline docs/guide/writing-tasks.md]: Quarts -> Quartz?

**Pull request merged**